### ISSUE TITLE
Using 'Not'

### ISSUE THREAD

**Issue by benjaminlees** [foundersandcoders/charter#1]
HI @sofer  I think it is good but I am not sure about the points which start 'not to' it can come off as a bit controlling,
An alternative might be' When applying for a position to first give fac an opportunity ....'

**Comment by NataliaLKB** [comment]
:+1:

**Comment by joshpitzalis** [comment]
I completely agree. Starting sentences with what not-to-do is confusing.  A possible rewrite here: https://github.com/foundersandcoders/charter/issues/4

**Comment by sofer** [comment]
@benjaminlees Created a pull request. https://github.com/foundersandcoders/charter/pull/6

Could you check and merge and then close this issue if you are happy with it. Cheers.

**Comment by nelsonic** [comment]
Respectfully, I disagree concerning using the word "_not_" in a charter.
If you read any specification e.g for a web protocol, the use of "specific" terms e.g: "must" or "must not" appear frequently in order to avoid any "interpretation" issues or "confusion".

## tl;dr

If we all have a common understanding that the _mission_ is to further the aim of "**_Providing Worldclass Team-based Technical Learning for Free at the point of access**_" (or words to that effect)  and that part of ensuring the _sustainability_ of the mission is securing employment through the program, then seeking employment without any consideration accruing to the program undermines the whole thing. So... Unless it's a precise rule, it will continue to be "forgotten"...

Let me make it clear that I am not a huge fan of the placement of alumni model,
However it has been the _fuel_ in the FAC engine that has kept the doors open and lights on till now.

And if we were _clear_ from the outset that in exchange for a _FREE_ education people agree to 
Let FAC "broker" the employment (_instead of an external recruiter who will most likely not be funding a learning mission_...) ... Perhaps we should make it clearer that **_we can place anyone anywhere**_ they chose, we are just as good at negotiating as any recruiter and given our technical competence, can spot when an opportunity might be promising more than it will deliver.
